tools/createNewRollup/createNewRollup.ts

```diff
@@ -0,0 +1,399 @@
+/* eslint-disable no-await-in-loop, no-use-before-define, no-lonely-if */
```

Comment:
Create outputs based on timestamps so they are not overwritten.
Similar to https://github.com/0xPolygonHermez/zkevm-contracts/blob/v9.0.0-rc.3-pp/tools/addRollupType/addRollupType.ts#L15

add output files to `.gitignore`